user/pages/03.commerce2/02.developer-guide/12.recurring/01.subscriptions-overview/docs.md

```diff
@@ -0,0 +1,69 @@
+
+![Fixed_annual_billing_schedule](../images/Fixed_annual_billing_schedule.png)
+
+The primary function of a billing schedule plugin is to generate billing periods:
+`generateFirstBillingPeriod()` generates the first billing period for a subscription, given a start date.
+`generateNextBillingPeriod()` generates subsequent billing periods for a subscription, given a start date and the current billing period.
+
+### What is a billing period?
+A billing period is an object that represents a span of time. Examples:
+ * - Oct 14th 14:56:20 - Nov 14th 14:56:20
+ * - May 1st 00:00:00 - June 1st 00:00:00
+ * - June 1st 00:00:00 - July 1st 00:00:00
+
+Billing periods are contiguous and represent [half-open ranges](http://wrschneider.github.io/2014/01/07/time-intervals-and-other-ranges-should.html) (the end date is not included in the duration).
+
+#### Billing period field type
+The Commerce Recurring module defines a custom "Billing Period" field type that is used by "recurring" type orders and "recurring" type order items. The field has two properties, the Start date and the End date, both of which are timestamps. A custom field widget and formatter are also provided for the billing period field type.
+
+### Prorater plugin
+In addition to the billing schedule plugin, the billing schedule configuration object also has a prorater plugin. Two prorater plugins are included with the core code: Full-Price and Proportional. Given an order item, a partial billing period, and a full billing period, the `prorateOrderItem()` method of the prorater plugin will calculate the prorated price. If a billing schedule uses the Full Price plugin, the order item's full price is simply returned without regard to the billing periods. If a billing schedule uses the Proportional plugin and the duration of the two billing periods is not the same, then the order item's price is adjusted by the duration of the subscription's usage.
+
+### Dunning
+Cron jobs are used to handle the processing of payments; the dunning settings affect how declined payments will be handled for the recurring order. How many retries? How much time between retries? What happens if we reach the maximum number of retries without success?
+![billing_schedule_dunning-1](../images/billing_schedule_dunning.png)
+
+## Subscription type plugin and Charges
+The Subscription type plugin serves as the bundle plugin type for the Subscription entity. The Commerce Recurring module provides two default subscription type bundles: the Product variation type and the Standalone type. The key method implemented by these plugins is the `collectCharges()` method, which returns an array of `Charge` objects for a subscription and a billing period. 
```

Comment:
We should not be describing the subscription type before showing the data model of the subscription, and describing when/how a subscription is created.